jwt/jwt.c

```diff
@@ -85,6 +85,16 @@ int jwt_verify(char * jwt,  point_t * pub_key)
 {
     ecc_init(); // TODO: Has to be initialized only once -> move to broker init phase
 
+    int dot_count = 0;
+    for (char *ptr = jwt; *ptr != '\0'; ptr++) {
```

Comment:
This is not an efficient solution.  We scan the whole string and count the dots. And then we do it again. 
Anyway, revert these changes. There is another story for them: IR-124.